<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerformanceWidget</class>
 <widget class="QWidget" name="PerformanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="2">
    <widget class="QTabWidget" name="tab1">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Returns and Drawdowns</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="PerformanceCanvas" name="canvas" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="tw_stats">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>2</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Correlations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QRadioButton" name="rb_daily">
         <property name="text">
          <string>Daily</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="rb_monthly">
         <property name="text">
          <string>Monthly</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="CorrelationChartCanvas" name="correlationCanvas" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="b_add">
     <property name="text">
      <string>Add...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="b_remove">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="lw_strategies">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PerformanceCanvas</class>
   <extends>QWidget</extends>
   <header>ui/performancecanvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CorrelationChartCanvas</class>
   <extends>QWidget</extends>
   <header>ui/correlationchartcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>b_add</sender>
   <signal>clicked()</signal>
   <receiver>PerformanceWidget</receiver>
   <slot>addResults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_remove</sender>
   <signal>clicked()</signal>
   <receiver>PerformanceWidget</receiver>
   <slot>removeResults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_daily</sender>
   <signal>toggled(bool)</signal>
   <receiver>PerformanceWidget</receiver>
   <slot>dailyToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_monthly</sender>
   <signal>toggled(bool)</signal>
   <receiver>PerformanceWidget</receiver>
   <slot>monthlyToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>572</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addResults()</slot>
  <slot>removeResults()</slot>
  <slot>dailyToggled(bool)</slot>
  <slot>monthlyToggled(bool)</slot>
 </slots>
</ui>
